### PR TITLE
add slider-pro v1.2.3 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/bqworks/slider-pro
**Watch 31
  Star 312
 Fork 134**
@Piicksarn  could u help me check this PR for #6387 ? thank you!

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/slider-pro/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "slider-pro",
+    "responsive-slider"
+  ],
+  "license": "MIT",
+  "npmName": "slider-pro",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.+(js|css|gif)"
```
Comment (PeterDaveHello) [inline ajax/libs/slider-pro/package.json]: Use "**/*" here.

**Comment by jamierocks** [comment]
:+1:

**Comment by maruilian11** [comment]
@PeterDaveHello i have added the `.cur` files.  please take a look.

**Comment by PeterDaveHello** [comment]
@maruilian11 ask others first.

**Comment by maruilian11** [comment]
@Piicksarn could you help me check the PR? thank you!

**Comment by Piicksarn** [comment]
@maruilian11, the pr is fine but rebase to master is required.
Would you ping me again after rebasing to master?
Thank you and sorry for the delay...!

**Comment by maruilian11** [comment]
@Piicksarn i have rebased it ~

**Comment by maruilian11** [comment]
@Piicksarn i have rebased the commit~

**Comment by jamierocks** [comment]
Massive thanks to @maruilian11 ;)

**Comment by maruilian11** [comment]
@jamierocks :+1:

**Comment by Piicksarn** [comment]
@PeterDaveHello, would you please have a look at the pr?
I think it's find now!
Thank you !

**Pull request merged**